doc/include/api_introspection.inc

```diff
@@ -1,144 +1,95 @@
 
-[para] This enables the Tcl code of a critcl-based package to
-distinguish between it getting used as a prebuilt package, versus
-dynamic compile & run, and take action based on that.
-
-[para] [emph Note] that this command is only useful from within a
-[file .critcl] file. The result is managed on a per-file basis, like
-is done for the commands embedding C code and controlling the
-behaviour of compiler and linker.
-
-[para] See also section [sectref {Modes Of Operation/Use}].
+[para] See also [sectref {Modes Of Operation/Use}].
 
 [comment ---------------------------------------------------------------------]
 [call [cmd ::critcl::failed]]
 
-This command returns a boolean value. It returns [const true] if
-critcl has failed to build the package, and [const false] otherwise,
-i.e. success. As part of this it forces the building of the package,
-but not its loading. Note that it will attempt to build the package
-only on the first call; future calls for the same package will return
-a cached result.
-
-[para] This enables a critcl-based package to check itself for
-availability and throw an error if it could not be built. [emph Note]
-that the command does [emph not] throw such an error itself.
+Returns [const true] the [vset critcl_script] could not be built, and
+[const false] otherwise.  Forces the building of the package if it hasn't
+already been done, but not its loading.  Thus, a [vset critcl_script] can
+check itself for availability of the compiled components.  Only useful from
+within a [vset critcl_script].
 
-[para] [emph Note] further that this command is only useful from
-within in a [file .critcl] file. The result is managed on a per-file
-basis, like is done for the commands embedding C code and controlling
-the behaviour of compiler and linker.
 
 [comment ---------------------------------------------------------------------]
 [call [cmd ::critcl::load]]
 
-This command is like [cmd critcl::failed], except that it also forces
-the loading of the generated shared library, if it was built, and that
-its result has reversed sense. 
-
-[para] It returns [const true] if critcl succeeded in building and
-loading the package, and [const false] otherwise, i.e. build- or
-load-failure.
-
-[para] This enables a critcl-based package to to not only check itself
-for availability and throw an error if it could not be built, but also
-force an immediate load, circumventing the default behaviour, which is
-lazy. See also section [sectref {Runtime Behaviour}]. [emph Note] that
-the command does [emph not] throw any error itself.
-
-[para] [emph Note] further that this command is only useful from
-within in a [file .critcl] file. The result is managed on a per-file
-basis, like is done for the commands embedding C code and controlling
-the behaviour of compiler and linker.
+Lik [cmd critcl::failed] except that it also forces the loading of the
+generated shared library, and that it returns true on success and false on
```

Comment:
true / false -> `[const ...]`

Reply:
Done with commit 29380521d0873ea1dcc5f8cdff6742b20764473f (Branch `documentation` in my repo, not seen by the PR).